js_modules/dagster-ui/packages/ui-core/src/workspace/RepositoryLocationsList.tsx

```diff
@@ -29,21 +68,39 @@ export const RepositoryLocationsList = () => {
   }
 
   return (
-    <Table>
-      <thead>
-        <tr>
-          <th>Name</th>
-          <th>Status</th>
-          <th>Updated</th>
-          <th>Definitions</th>
-          <th>Actions</th>
-        </tr>
-      </thead>
-      <tbody>
-        {locationEntries.map((location) => (
-          <CodeLocationRowSet key={location.name} locationNode={location} />
-        ))}
-      </tbody>
-    </Table>
+    <>
+      <VirtualizedCodeLocationHeader />
+      <div style={{overflow: 'hidden'}}>
+        <Container ref={parentRef}>
+          <Inner $totalHeight={totalHeight}>
+            <DynamicRowContainer $start={items[0]?.start ?? 0}>
+              {items.map(({index, key}) => {
+                const row: CodeLocationRowType = codeLocations[index]!;
+                if (row.type === 'error') {
+                  return (
+                    <VirtualizedCodeLocationErrorRow
+                      key={key}
+                      data-index={index}
+                      locationNode={row.node}
+                      measure={rowVirtualizer.measure}
```

Comment:
I think this should be `measureElement` which is grabbed from `items`,

Reply:
Oh, interesting. The tanstack codesandbox example does what I've done here, and it does work. And the `measureElement` key does not appear to be on the item type in the codesandbox's TS definitions. (https://codesandbox.io/p/sandbox/github/tanstack/virtual/tree/beta/examples/react/dynamic?embed=1&file=%2Fsrc%2Fmain.tsx%3A84%2C1-85%2C1)

Maybe it's been removed in one of the more recent beta versions?

Reply:
That is example is using `measureElement` but looks like it was moved to the virtualizer?